misc/db_schema.yaml

```diff
@@ -194,7 +194,7 @@ availability:
     - { Field: availability_id, Type: 'int unsigned', 'Null': false, Extra: auto_increment }
     - { Field: device_id, Type: 'int unsigned', 'Null': false, Extra: '' }
     - { Field: duration, Type: bigint, 'Null': false, Extra: '' }
-    - { Field: availability_perc, Type: 'double(6,6)', 'Null': false, Extra: '', Default: '0.000000' }
+    - { Field: availability_perc, Type: 'float(9,6)', 'Null': false, Extra: '', Default: '0.000000' }
```

Comment:
This still needs to be type double.  remember DBAL = database abstraction layer :)

Reply:
a little bit confused, but if it's work i'm ok with ;-)